menus/sgd/processoption.cfg

```diff
@@ -17,14 +17,13 @@
 source "${sg2d_directory}/tools.cfg"
 
 for ncfg in ${sourced_cfgs} ; do
-
+  menu_name="${ncfg}"
```

Comment:
You can remove this assignment because this variable is set on the next line.

Reply:
It's the fallback value for the exotic case that the regexp mit not match. I'll clarify it.